src/main/java/com/cloudogu/smeagol/wiki/infrastructure/PullChangesStrategy.java

```diff
@@ -0,0 +1,19 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import com.cloudogu.smeagol.wiki.domain.WikiId;
+
+/**
+ * A {@link PullChangesStrategy} decides, if changes should be pulled from the remote repository of the wiki.
+ */
+public interface PullChangesStrategy {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Annotate the "PullChangesStrategy" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1609)